jobs/build.Jenkinsfile

```diff
@@ -442,50 +379,18 @@ lock(resource: "build-${params.STREAM}") {
-                // lower to make sure we don't go over and account for overhead
-                def xz_memlimit = cosa_memory_request_mb - 512
-                shwrap("""
-                export XZ_DEFAULTS=--memlimit=${xz_memlimit}Mi
-                cosa compress --compressor xz --artifact metal --artifact metal4k
-                """)
-                try {
-                    parallelruns = [:]
-                    parallelruns['metal'] = {
-                        shwrap("cosa kola testiso -S --output-dir tmp/kola-testiso-metal")
-                    }
-                    parallelruns['metal4k+uefi'] = {
-                        shwrap("cosa kola testiso -SP --qemu-native-4k --qemu-multipath --output-dir tmp/kola-testiso-metal4k")
-                        shwrap("cosa shell -- mkdir -p tmp/kola-testiso-uefi")
-                        shwrap("cosa kola testiso -S --qemu-firmware=uefi --scenarios iso-live-login,iso-as-disk --output-dir tmp/kola-testiso-uefi/insecure")
-                        shwrap("cosa kola testiso -S --qemu-firmware=uefi-secure --scenarios iso-live-login,iso-as-disk --output-dir tmp/kola-testiso-uefi/secure")
-                    }
-                    // process this batch
-                    parallel parallelruns
-                } catch (Throwable e) {
-                    throw e
-                } finally {
-                    shwrap("""
-                    cosa shell -- tar -c --xz tmp/kola-testiso-metal/ > kola-testiso-metal.tar.xz
-                    cosa shell -- tar -c --xz tmp/kola-testiso-metal4k/ > kola-testiso-metal4k.tar.xz
-                    cosa shell -- tar -c --xz tmp/kola-testiso-uefi/ > kola-testiso-uefi.tar.xz
-                    """)
-                    archiveArtifacts allowEmptyArchive: true, artifacts: 'kola-testiso*.tar.xz'
-
-                    // For now we want to notify ourselves when this workaround is observed. It won't
-                    // fail the build, just give us information.
-                    // https://github.com/coreos/fedora-coreos-tracker/issues/1233
-                    def grepRc = shwrapRc("""
-                         cosa shell -- grep 'tracker issue workaround engaged for .*issues/1233' \
-                            tmp/kola-testiso-uefi/insecure/{iso-live-login,iso-as-disk}/console.txt
-                    """)
-                    if (grepRc == 0) {
-                        warnError(message: 'Detected used workaround for #1233') {
-                            error('Detected used workaround for #1233')
-                        }
-                    }
+            // Run Kola TestISO tests for metal artifacts
+            kolaTestIso(cosaDir: env.WORKSPACE, build: newBuildID, arch: basearch)
+            // For now we want to notify ourselves when this workaround is observed. It won't
+            // fail the build, just give us information.
+            // https://github.com/coreos/fedora-coreos-tracker/issues/1233
```

Comment:
While we're here, let's add a `shwrap` here that just checks that the file we'll grep in the next `shwrapRc` actually exists.

Reply:
done

Reply:
From a test run the added `shwrap` shows:

```
19:25:36  + ls tmp/kolaTestIso-BJqWL/kola-testiso-uefi/insecure/iso-live-login/console.txt tmp/kolaTestIso-BJqWL/kola-testiso-uefi/insecure/iso-as-disk/console.txt
19:25:36  tmp/kolaTestIso-BJqWL/kola-testiso-uefi/insecure/iso-as-disk/console.txt
19:25:36  tmp/kolaTestIso-BJqWL/kola-testiso-uefi/insecure/iso-live-login/console.txt
```